pkg/storage/storage_test.go

```diff
@@ -54,28 +52,53 @@ var _ = Describe("In-memory storage", func() {
 
 	BeforeEach(func() {
 		now = time.Now()
+		prevTS := now.Add(-10 * time.Second)
+		prevBatch := &MetricsBatch{
+			Nodes: []NodeMetricsPoint{
```

Comment:
Could you add a test case in which:
* New node/pod is added that was missing in previous batch
* Node/pod is removed that was available in previous batch

Reply:
I added the tests